### PR TITLE
#155 using `alert` instead of `alarm`

### PR DESCRIPTION
**Pull request by dortegau**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions
* Remember to set **idealista:develop** as base branch;

### Description of the Change

Code base should use the ubiquitous term "Alert" instead of "Alarm".

### Benefits

Improves code symmetry and uses ubiquitious language

### Possible Drawbacks

Afaik there aren't (I ran unit tests and e2e tests and works fine)

### Applicable Issues

#155

### CODE REVIEW

**Review by jperera — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -145,7 +146,7 @@ Now connector field is mandatory in helm chart is mantatory.
 ## [2.2.1](https://github.com/idealista/prom2teams/tree/2.2.1)
 [Full Changelog](https://github.com/idealista/prom2teams/compare/2.1.2...2.2.1)
 ## Added
-- *[#80](https://github.com/idealista/prom2teams/pull/79) Add the possibility of group alarms by alertname* @manuhortet
```
Comment (blalop) [inline CHANGELOG.md:148]: Should we also update the changelog of already released versions?

File: prom2teams/app/sender.py [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
-            alarms = map_prom_alerts_to_teams_alarms(alerts)
-        return self.json_composer.compose_all(alarms)
+            alerts = map_prom_alerts_to_teams_alerts(alerts)
+        return self.json_composer.compose_all(alerts)
 
-    def send_alarms(self, alerts, teams_webhook_url):
-        sending_alarms = self._create_alarms(alerts)
-        for team_alarm in sending_alarms:
-            self.teams_client.post(teams_webhook_url, team_alarm)
+    def send_alerts(self, alerts, teams_webhook_url):
```
Comment (blalop) [inline prom2teams/app/sender.py:23]: Is out of scope in this PR, but in order to bring discussion with this with the rest of the team: maybe we should start talking about Teams cards/messages here, as it's a term that fits better with what we are sending - in fact, everything that json_composer touch becomes a Teams card/message

**Review by blalop — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (dortegau) [reply CHANGELOG.md:148]: I would rather use the same name to refer to the same concept. In that sentence "alarms" and "alerts" are mixed.

File: prom2teams/app/sender.py (hunk 2 above)
Comment (dortegau) [reply prom2teams/app/sender.py:23]: I think it would be a great idea to create an issue to discuss it

**Review by jmonterrubio — APPROVED** [review]

**Review by caldito — APPROVED** [review]

**Review by vicsufer — APPROVED** [review]

**Pull request merged**